TileFormats/Geometry/README.md

```diff
@@ -0,0 +1,110 @@
+* If `ELLIPSOIDS_LENGTH` in not defined, or zero, no ellipsoids will be rendered.
+* If `SPHERES_LENGTH` in not defined, or zero, no spheres will be rendered.
+Multiple feature types may be defined in a single Geometry tile using multiple `LENGTH` semantics, and in that case, all specified feature types will be rendered.
+
+If a semantic has a dependency on another semantic, that semantic must be defined as well.
+
+#### Geometry Semantics
+
+| Semantic | Data Type | Description | Required |
+| --- | --- | --- | --- |
+| `BOXES` | `float32[]` | The boxes in the tile. The length of the array will be `16 * BOXES_LENGTH`. The 16 elements are the 4x4 model matrix in column-major order. | :white_check_mark: Yes, when the global `BOXES_LENGTH` is greater than zero. |
+| `BOX_BATCH_IDS` | `uint16[]` | The `batchId` of the box that can be used to retrieve metadata from the `Batch Table`. | :red_circle: No. |
+| `CYLINDERS` | `float32[]` | The cylinders in the tile. The length of the array will be `16 * CYLINDERS_LENGTH`. The 16 elements are the 4x4 model matrix in column-major order. The scale of the x and y axis will be the radius of the cylinder. Scaling the z axis will change the length of the cylinder. | :white_check_mark: Yes, when the global `CYLINDERS_LENGTH` is greater than zero. |
+| `CYLINDER_BATCH_IDS` | `uint16[]` | The `batchId` of the cylinder that can be used to retrieve metadata from the `Batch Table`. | :red_circle: No. |
+| `ELLIPSOIDS` | `float32[]` | The ellipsoids in the tile. The length of the array will be `16 * ELLIPSOIDS_LENGTH`. The 16 elements are the 4x4 model matrix in column-major order. | :white_check_mark: Yes, when the global `ELLIPSOIDS_LENGTH` is greater than zero. |
+| `ELLIPSOID_BATCH_IDS` | `uint16[]` | The `batchId` of the ellipsoid that can be used to retrieve metadata from the `Batch Table`. | :red_circle: No. |
+| `SPHERES` | `float32[]` | The spheres in the tile. The length of the array will be `3 * SPHERES_LENGTH`. The 3 elements are the translation of the sphere. | :white_check_mark: Yes, when the global `SPHERES_LENGTH` is greater than zero. |
+| `SPHERE_BATCH_IDS` | `uint16[]` | The `batchId` of the sphere that can be used to retrieve metadata from the `Batch Table`. | :red_circle: No. |
+
+#### Global Semantics
+
+The semantics define global properties for all geometry elements.
+
+| Semantic | Data Type | Description | Required |
+| --- | --- | --- | --- |
+| `BOXES_LENGTH` | `uint32` | The number of boxes in the tile. | :red_circle: No. |
+| `CYLINDERS_LENGTH` | `uint32` | The number of cylinders in the tile. | :red_circle: No. |
+| `ELLIPSOIDS_LENGTH` | `uint32` | The number of ellipsoids in the tile. | :red_circle: No. |
+| `SPHERES_LENGTH` | `uint32` | The number of spheres in the tile. | :red_circle: No. |
+| `RTC_CENTER` | `float32[]` | The center used for RTC rendering. | :red_circle: No. If no center is supplied, the center of the global `RECTANGLE` will be used. |
```

Comment:
`RECTANGLE` is not included in `GEOM`.

Reply:
`RECTANGLE` and `REGION` aren't required by `geom`.

Reply:
What I mean is, `RECTANGLE` should be removed from the `RTC_CENTER` description because it doesn't exist for the `geom` format.

Reply:
Ah, I missed that. Thanks.